src/shared/types/api/tokenValidator.ts

```
@@ -0,0 +1,42 @@
+import { axiosClient } from './http-client';
+import useAuthStore from '../../../app/provider/authStore';
+
+// 타입 가드 함수
+function isTokenError(error: unknown): error is { code: string; status?: number } {
+  return error !== null && typeof error === 'object' && 'code' in error && typeof (error as any).code === 'string';
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)